app/scripts/shell/dimAppCtrl.controller.js

```diff
@@ -231,15 +227,16 @@
               classType: -1,
               name: 'random',
               items: items
-            })
+            });
           }
+          return $q.reject();
```

Comment:
this function was expecting a return statement... so i added `$q.reject()` that acceptable?